src/sentry/issues/endpoints/group_first_last_release.py

```diff
@@ -25,13 +26,20 @@ class GroupFirstLastReleaseEndpoint(GroupEndpoint):
         }
     )
 
-    def get(self, request: Request, group) -> Response:
-        """Get the first and last release for a group.
+    def get(self, request: Request, group: Group) -> Response:
+        """
+        Get the first and last release for a group, within the environments provided.
+        If no environments are provided, the data returned will be across all environments.
 
         This data used to be returned by default in group_details.py, but now that we
         can collapse it, we're providing this endpoint to fetch the data separately.
         """
-        first_release, last_release = get_first_last_release(request, group)
+
+        environment_names = list(set(request.GET.getlist("environment", default=[])))
```

Comment:
can we use the helper method `get_environments` instead? https://github.com/getsentry/sentry/blob/ca40c14646767fdcd06dd90a8089c116bd8f6cde/src/sentry/api/helpers/environments.py#L48

Reply:
yeah, fair point. sounds good, will update that